gsa/public/locales/gsa-de.json

```diff
@@ -1659,7 +1659,7 @@
   "The maximum of {{num}} hosts was exceeded. If there are more hosts associated with this process, they will not be taken into account.": "Das Maximum von {{num}} Hosts wurde überschritten. Sollten weitere Hosts mit diesem Prozess verknüpft sein, werden sie nicht mitberücksichtigt.",
   "The name must include at least one alphanumeric character or one of .,-/_# and space.": "Der Name muss mindestens ein alphanumerisches Zeichen oder eines von .,-/_# und Leerzeichen enthalten.",
   "The port range needs numerical values for start and end!": "Der Portbereich benötigt numerische Werte für Start und Ende!",
-  "The report cannot be displayed because no Greenbone Vulnerability Manager report format is available. This could be due to a missing feed. Please update the feed, check the \"feed import owner\" setting, or contact your system administrator.": "Der Bericht kann nicht angezeigt werden, weil dem Greenbone Vulnerability Manager kein Berichtformat zur Verfügung steht. Dies könnte an einem fehlenden Feed liegen. Bitte aktualisieren Sie den Feed, überprüfen die \"Feed Import Besitzer\"-Einstellungen, oder kontaktieren Ihren Systemadministrator.",
+  "The report cannot be displayed because no Greenbone Vulnerability Manager report format is available. This could be due to a missing GVMd data feed. Please update the GVMd data feed, check the \"feed import owner\" setting, or contact your system administrator.": "Der Bericht kann nicht angezeigt werden, weil dem Greenbone Vulnerability Manager kein Berichtformat zur Verfügung steht. Dies könnte an einem fehlenden GVMd Data Feed liegen. Bitte aktualisieren Sie den GVMd Data Feed, überprüfen die \"Feed Import Besitzer\"-Einstellungen, oder kontaktieren Ihren Systemadministrator.",
```

Comment:
```suggestion
  "The report cannot be displayed because no Greenbone Vulnerability Manager report format is available. This could be due to a missing GVMd data feed. Please update the GVMd data feed, check the \"feed import owner\" setting, or contact your system administrator.": "Der Bericht kann nicht angezeigt werden, weil dem Greenbone Vulnerability Manager kein Berichtformat zur Verfügung steht. Dies könnte an einem fehlenden gvmd Data Feed liegen. Bitte aktualisieren Sie den gvmd Data Feed, überprüfen die \"Feed Import Besitzer\"-Einstellungen, oder kontaktieren Ihren Systemadministrator.",
```

Reply:
This `GVMd Data Feed` was taken from the text introduced in https://github.com/greenbone/gvmd/pull/1155. Will change it here and create a PR for gvmd to fix the spelling there as well.

Reply:
Made the `GVMd` on manger side lowercase in https://github.com/greenbone/gvmd/pull/1236 and in the text here with 33fb63f4448827ce5c8a00441679cd96be8affa6